app/actuators/ReactiveElement.js

```diff
@@ -295,7 +295,7 @@ export default class ReactiveElement extends Component {
 			"rbc-resultstats-inactive": !this.props.showResultStats,
 			"rbc-noresults-active": this.props.noResults,
 			"rbc-noresults-inactive": !this.props.noResults
-		});
+		}, this.props.className);
```

Comment:
In actuators, we need to apply `this.props.className` at the root level. Here, `cx` doesn't get appended to the classes of the outermost container.

Also, support for `style` prop is missing here.